### DISCUSSION TITLE
Summits'

### DISCUSSION THREAD

**Discussion by aminoxix** [EddieHubCommunity/support#1443] [General] [answered]
Great to see you here! 👋🏼
- 

Some amazing **Developer Summits**  are flowing freely, its a moment for us to lookout some amazing upcoming as well as passed summits, to empower communities with enhance knowledge apart and Grab this opportunity to make it real-time valuable for hubbers..

Love to mention, some of them as follows :

|   UPCOMING SUMMITS  |  SCHEDULE  |
|  -------  | -------- |
| <a href="https://aws.amazon.com/events/summits/online/?awsf.events-location=*all&awsf.events-series=*all"> AWS Global Summit Online </a>  |  2021 |
| <a href="https://www.cloudfoundry.org/events/summit/"> Cloud Foundry Summit Virtual </a> |  July 21-22, 2021 |
| <a href="https://studentsummit21.eventbrite.com/?aff=Avinash"> Postman Student Summit </a> | August 07-08, 2021 |
| <a href="https://summit.riot-os.org/2021/"> RIOT Summit </a> | September 9-10, 2021 |
| <a href="https://1point21gws.com/developersummit/bangalore/"> World Developers Summit - Bangalore | September 16-17, 2021 |
| <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/"/> KubeCon + CloudNativeCon North America 2021 Virtual | October 12 – 15, 2021 |
| <a href="https://www.israelmobilesummit.com/"> Israel Mobile Summit </a> | Oct. 27, 2021 |

|  PASSED SUMMITS |  SCHEDULED |
|  --------------- | -------------- |
|  <a href="https://developer.apple.com/"> Apple WWDC21 </a>  | June 2021  |
| <a href="https://events.linuxfoundation.org/gitops-summit/"> GitOps Summit - cdCon </a> | June 2021 |
| <a href="https://geekle.us/flutter"> Flutter Global Summit'21 </a> | June 2021 |
| <a href="https://www.jenkins.io/events/contributor-summit/"> Jenkins Contributor Summit </a> | June 2021 |
| <a href="https://www.hyperledger.org/"> Hyperledger Global Forum </a> | June 2021 |
| <a href="https://globalmaintainersummit.github.com/" > Global Github Maintainer Summit </a> | June 2021 |
| <a href="https://wiki.freebsd.org/DevSummit"> FreeBSD Developer & Vendor Summits </a> | June 2021 |
| <a href="https://g.co/appliedmlsummit"> Google Cloud - Applied ML Summit </a> | June 2021 |
| <a href="https://tokyo-2021.devrel.net/"> DevRelCon Tokyo | June 2021 |
|  <a href="https://developers.facebook.com/f8/"> F8 Refresh </a>  | June 2021  |
| <a href="https://mybuild.microsoft.com/home"> MS Build </a> | May 2021 | 
|  <a href="https://www.docker.com/dockercon-live/2021/"> DockerCon </a>  | May 2021  |
|  <a href="https://deepsource.io/next/"> DeepSource Next ’21 </a>  | 2021  |
| <a href="https://events.google.com/io/?lng=en"> Google I/O </a> | 2021 |
|  <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/"> KubeCon + CloudNativeCon Europe 2021 Virtual </a> | 2021 |
| <a href="https://rasa.com/summit/"> The RASA Summit </a> | 2021 |
| <a href="https://www.redhat.com/en/summit"> RedHat Summit </a> | 2021 |
| <a href="https://mlds.analyticsindiasummit.com/"> Machine Learning Developers Summit (Paid) </a>| 2021 |
| <a href = "https://business.adobe.com/summit/adobe-summit.html"> Adobe Summit </a> | 2021 | 
| <a href="https://developer.chrome.com/devsummit/"> Chrome Dev Summit </a>  |  2020 |
| <a href="https://firebase.google.com/summit"> Firebase Dev Summit </a> | 2020 |
| <a href="https://events.withgoogle.com/gdc2020/"> Google for Games Developer Summit </a> | 2020 |
| <a href="https://webinar.intel.com/oneAPIDeveloperSummit2020"> oneAPI Developer Summit </a> | 2020 |
| <a href="https://developer.android.com/dev-summit"> Android Dev Summit </a>  | 2019 |
| <a href="https://aspirationtech.org/events/devsummit19"> Nonprofit Software Development Summit </a> | 2019 |
| <a href="https://www.mediawiki.org/wiki/Wikimedia_Developer_Summit"> Wikipedia Developer Summit </a> | 2018 |
| <a href="https://geonode.org/geonode-summit-2016/about/"> GeoNode Summit 2016 </a> | 2016 |
| <a href="https://openzfs.org/wiki/OpenZFS_Developer_Summit"> OpenZFS Developer Summit </a> | Regular |
| <a href="https://wiki.ubuntu.com/DeveloperSummit"> Ubuntu Developer Summit </a> | Previous |
| <a href="http://www.developermarch.com/developersummit/2013/report/presentations.html"> Great Indian Developer Summit </a> | Previous |
| <a href="https://openjdk.java.net/projects/mlvm/jvmlangsummit/"> JVM Language Summit </a> | Previous |

**NOTE** : Events are scheduled with dates as above mentioned, register before event getting start. 

[![](https://img.shields.io/badge/</>%20With%20❤️%20By-AMINOS-green)](https://github.com/amino19)

**Comment by kaiwalyakoparkar** [comment]
I appreciate your work

**Comment by Prathamesh-Shanbhag** [comment, marked as answer]
Thank you for sharing about this! 🔥

**Comment by kaiwalyakoparkar** [comment]
@amino19 Could you please provide the sources of this information. I mean where did you get this information from?

**Reply by aminoxix** [reply]
Is it compulsory to share the bibliography?
Like, some of the summits; I attended earlier, some are suggested from mine colleagues & some are listed from open search..
Hope you understand :)

**Reply by kaiwalyakoparkar** [reply]
No.. No problem. I was just curious after seeing all of them. I would love to see more :heart:

**Reply by aminoxix** [reply]
Don't worry, I'll update this regularly.. <3

**Reply by kaiwalyakoparkar** [reply]
Thanks for that :heart:

**Comment by rakesh0x7** [comment]
@amino19 you can the **MS BUILD** to the list which was going to happen on May 25–27, 2021
you can refer to this [link](https://mybuild.microsoft.com/home)

**Reply by aminoxix** [reply]
Wow, that's really amazing to see you here, @deephunt3r! 
Thanks for your kind suggestion!

Hope we'll increase our summits' family! <3

**Comment by avinashupadhya99** [comment]
Hey, cdCon is happening right now - https://events.linuxfoundation.org/cdcon/register/ :)

**Reply by aminoxix** [reply]
Yes, I'm attending that! 
- GitOps Summit :flags:

**Reply by kaiwalyakoparkar** [reply]
Awesome. Thanks for sharing

**Comment by avinashupadhya99** [comment]
Hey folks, the Postman Student Summit is happening next month on August 07-08. You can sign up at https://studentsummit21.eventbrite.com/

If you’re interested in APIs, you should definitely check out the  #PostmanStudentSummit, a free API conference for students and educators around the world! The Summit will have talks delivered by students and educators, a keynote from Postman CEO Abhinav Asthana, and a pre-event data visualization hackathon.

Here's my affiliate link - https://studentsummit21.eventbrite.com/?aff=Avinash *(It's just a way to track how many folks each Student Leader got to sign up)* :)

**Reply by aminoxix** [reply]
@avinashupadhya99 : Ah, thanks for the info., even I'm a student expert!
Thanks for mentioning :slightly_smiling_face:

**Reply by aminoxix** [reply]
BTW, I will promote your link as well! 😉
